src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -131,42 +97,6 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
             }
         }
 
-
-        [Test]
-        public void Setter_dependencies_should_be_supported()
```

Comment:
>. So that means if for some reason a container forgets to support setter injection existing code like

Since the core no longer require property injection, why would we have tests for it? (that test suite is to verify core assumptions around the container abstraction)

If a container want to support it that container adapter should have tests for it? (what if someone wants to use a container that doesn't support property injection)